tests/contest/runtimetest/src/tests.rs

```diff
@@ -546,6 +546,19 @@ pub fn test_io_priority_class(spec: &Spec, io_priority_class: IOPriorityClass) {
     }
 }
 
+pub fn test_validate_root_readonly() {
+    if let std::io::Result::Err(e) = test_read_access("/") {
+        let errno = Errno::from_raw(e.raw_os_error().unwrap());
+        if errno == Errno::ENOENT {
+            /* This is expected */
+        } else {
+            eprintln!("in readonly paths, error in testing read access for / : {e:?}");
+        }
+    } else {
+        /* Expected */
+    }
+}
```

Comment:
I think this logic is wrong - we need to check if write access is false, not read-access is giving any error or not. Also if root is readonly , why would the read access give any error at all ? It should succeed, even ENOENT is not expected.